cms/djangoapps/contentstore/views/item.py

```diff
@@ -84,6 +84,12 @@
 
 
 highlights_setting = WaffleSwitch(u'dynamic_pacing', u'studio_course_update')
+WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name=u'test_settings')
+SHOW_REVIEW_RULES_FLAG = CourseWaffleFlag(
```

Comment:
I recall there being a helper method (possibly in waffle_utils?) which pulls up this object and checks whether it's enabled, returning a boolean. I may be missing something, but I don't see a benefit to using this more low-level approach (i.e. constructing an ORM representation and then checking its attributes) over the simple "give me the name, I'll return a boolean" api I think exists somewhere out there.

Reply:
This seems to be the common pattern throughout.

Reply:
You're right; was getting mixed up b/t switches & flags